build/vkd3d_version.c

```diff
@@ -0,0 +1 @@
+const char vkd3d_build[] = "v1.1";
```

Comment:
Is this a leftover from Meson PR?

Reply:
oops, appears to be!

Reply:
Could you also please add the "Signed-off-by:" line, to be consistent with other commits.

Reply:
Done